providers/defltprov.c

```diff
@@ -320,6 +320,10 @@ static const OSSL_ALGORITHM deflt_macs[] = {
 
 static const OSSL_ALGORITHM deflt_kdfs[] = {
     { PROV_NAMES_HKDF, "provider=default", ossl_kdf_hkdf_functions },
+#ifndef OPENSSL_NO_TLS1_3
+    { PROV_NAMES_TLS1_3_KDF, "provider=default",
+      ossl_kdf_tls1_3_prf_functions },
```

Comment:
It is a little bit inconsistent that sometimes there is `tls1_3_prf` and elsewhere `TLS1_3_KDF`.